storage/object/how-to/use-obj-stor-with-private-networks.mdx

```diff
@@ -44,18 +44,18 @@ Follow the instructions for [creating an Instance](/compute/instances/how-to/cre
 
 ## How to create a Private Network and attach the Instance
 
-1. Follow the instructions for [creating a Private Network](/network/vpc/how-to/create-private-network/). Make sure you create it in the region that encompasses the Availability Zone of the Instance you previously created.
+1. Follow the instructions for [creating a Private Network](/network/vpc/how-to/create-private-network/). Make sure you create it in the same region of the Availability Zone of the Instance you previously created.
```

Comment:
I think I find the original clearer, but maybe it's personal preference?  Or perhaps "...in the region that contains/includes the Availability Zone" ?